libs/vkd3d/device.c

```diff
@@ -4432,21 +4432,20 @@ static D3D12_RAYTRACING_TIER d3d12_device_determine_ray_tracing_tier(struct d3d1
     unsigned int i;
 
     /* Tier 1.0 formats. 1.1 adds:
-     * - RGB8_{U,S}NORM
+     * - RGBA8_{U,S}NORM
      * - RG16_UNORM
-     * - RGB16_UNORM
+     * - RGBA16_UNORM
      */
     static const VkFormat required_vbo_formats[] = {
         VK_FORMAT_R32G32_SFLOAT,
         VK_FORMAT_R32G32B32_SFLOAT,
         VK_FORMAT_R16G16_SFLOAT,
         VK_FORMAT_R16G16_SNORM,
-        /* DXR specifies RGBA16 here, but it also completely ignores A,
-         * so it's *actually* DXGI_FORMAT_R16G16B16_FLOAT/SNORM,
-         * but those formats don't exist in D3D,
-         * and they couldn't be bothered to add those apparently <_<. */
-        VK_FORMAT_R16G16B16_SFLOAT,
-        VK_FORMAT_R16G16B16_SNORM,
+        /* DXR specifies RGBA16 here, but it also completely ignores A.
+         * So does Vulkan, and you can technically load out of bounds with this,
+         * but behavior in DXR and Vulkan is actually supposed to match here, so ... */
```

Comment:
Do we need this comment anymore? It's a 1:1 mapping with no real quirks attached.

Reply:
Fixed.